docs/UserGuide_CS2101.md

```diff
@@ -15,13 +15,13 @@ In order to help speed up your workflow, we have created commands for tasks you
 | Add a new student                                                                    | [`add` command](#adding-a-student-add)                                                          |
 | Edit a student's details (phone number, address for example)                         | [`edit` command](#editing-student-details-edit)                                                 |
 | Mark a student's attendance and remind yourself to collect payment from the student. | [`mark` command](#marking-a-student-mark)                                                       |
-| Receive money from a student                                                          | [`pay` command](#receiving-money-from-a-student-pay)                                            |
-| Know the students you have                                                           | [`list` command](#viewing-all-students-list)                                                    |
+| Receive money from a student                                                         | [`pay` command](#receiving-money-from-a-student-pay)                                            |
+| View the students under you                                                          | [`list` command](#viewing-all-students-list)                                                    |
 | Search for student's information                                                     | [`find` command](#finding-a-student-find) / [`sort` command](#sort-the-displayed-students-sort) |
 | Find next available class for student                                                | [sequence of commands](#next-available-class)                                                   |
 | Set a student's next class                                                           | [`edit` command](#editing-student-details-edit)                                                 |
 | Taking notes                                                                         | [`edit` command](#editing-student-details-edit)                                                 | 
-| Remove a student as he is no longer a student                                        | [`delete` command](#deleting-students-delete)                                                   |
+| Remove a student as he quits the tuition                                             | [`delete` command](#deleting-students-delete)                                                   |
```

Comment:
Quits the tuition sounds a bit weird. I think maybe you can replace it with "as he stopped tuition"